src/main/main.js

```diff
@@ -2,7 +2,7 @@
 'use strict';
 import configureStore from 'common/configure-store';
 import { localeUpdate } from 'common/locale/actions';
-import Logger from 'common/logger';
+const { Logger } = require('common/logger');
```

Comment:
Why not use the `import` syntax here?

Reply:
replaces `import Logger from 'common/logger';`  with `const { Logger } = require('common/logger');`
will fix for that file

Reply:
in general mixing thouis two styles are a very bad practice. Added fixing it for the whole project as subtask for #268